docs/2-protocol-overview/fees.md

```diff
@@ -0,0 +1,39 @@
+---
+sidebar_position: 4
+---
+
+# Fees
+
+:::caution
+
+This article may contain temporary information and applies only to the ErgoDEX V1
+
+:::
+
+ErgoDEX protocol V1 includes 3 types of fees:
+
+| Fee              | Assignment                         | Unit                              | Amount                            |
+| ---------------- | ---------------------------------- | --------------------------------- | --------------------------------- |
+| Network Fee      | Paid to Miners or Node stake pools | Native network currency (ERG/ADA) | min - ∞ (user choice)             |
+| Execution Fee    | Paid to execution bots             | Native network currency (ERG/ADA) | [Formula](#execution-fee-formula) |
+| DEX Fee (UI Fee) | Paid to UI provider                | Native network currency (ERG/ADA) | 0.01 ERG / 0.4 ADA                |
+
+#### Execution fee formula
+
+**Execution Fee** is always a range of values (`minExFee - maxExFee`)
+
+`minExFee = minerFee * 1.5`
+
+`1.5`: constant factor which have to satisfy the condition `minExFee > minerFee`
+
+`maxExFee = minExFee * Nitro`
+
+#### Nitro
```

Comment:
Why is "Nitro" duplicated here and in the glossary section?

Reply:
@oskin1 
It has to be titled so users can easily share this info with each other.
Will change to "What is Nitro?"